### ISSUE TITLE
GUI text is being truncated on menus and guide channel names on arm64-v8a but not armeabi-v7

### ISSUE THREAD

**Issue by mmortal03** [xbmc/xbmc#23561]
### Describe the bug

GUI text is being truncated on menus and guide channel names on arm64-v8a but not armeabi-v7.
I'm seeing this on Android 11 64-bit (crdroid) in Kodi 20.2, Kodi 21 Alpha 2, and the latest master nightly.

### To Reproduce
1. Install the Kodi apk
2. Load Kodi
3. Note the truncated text in the GUI

### Debuglog

The debuglog can be found here:
[kodi.log](https://github.com/xbmc/xbmc/files/12194146/kodi.log)

### Screenshots 

![Screenshot_20230728-063341715](https://github.com/xbmc/xbmc/assets/8298277/76278bdc-89ba-4562-9b16-55c8bf53676b)

### Your Environment
Used Operating system:

 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Android 11 64-bit (crdroid)
 - Kodi version:  Kodi 20.2, Kodi 21 Alpha 2, and the latest master nightly.

**Comment by mmortal03** [comment]
This is even occurring in file and directory names in the File manager. Changing the font in Estuary to Arial doesn't fix the problem.

**Comment by enen92** [comment]
@mmortal03 can you give it a shot to current nightlies?

**Comment by mmortal03** [comment]
@enen92 , yep, I already tested: kodi-20230726-e402e9bb-master-arm64-v8a.apk

**Comment by mmortal03** [comment]
@enen92 , It looks like this has been fixed somewhere along the line in the 22.0 alpha 64-bit development builds, as I can't reproduce it on the latest build. It *is* still present in the 21.0 Omega 64-bit official build, though. Should we leave this bug report open until 22.0 has been officially released? (I'm not sure what the protocol is here on when to close an issue.) 

Tangential, but the old phone that I've been testing this on, the battery just expanded, breaking open the edge of the case and bending the screen(!), and this only happened in the last few days. I'm glad this stale bug reminder happened when it did.
